tiledb/sm/cpp_api/config.h

```diff
@@ -281,6 +281,13 @@ class Config {
    *    (except parallel S3 writes, which are controlled by
    *    `vfs.s3.multipart_part_size`.) <br>
    *    **Default**: 10MB
+   * - `vfs.max_batch_read_size` <br>
+   *    The maximum number of bytes in a batched VFS read operation. <br>
+   *    **Default**: 100MB
+   * - `vfs.max_batch_read_amplification` <br>
+   *    The maximum amplification factor (>= 1.0) of batched VFS read
+   *    operations. <br>
+   *    **Default**: 1.5
```

Comment:
Should this be > 1 by default?

Reply:
Hmm, good question. Maybe 1 is safer.

Reply:
I think we should default to 1, with the nice optimization now that adjacent reads are now grouped together by default.